packages/tupaia-web/src/features/EnlargedDashboardItem/ExportDashboardItem.tsx

```diff
@@ -71,46 +63,6 @@ const Button = styled(BaseButton)`
   text-transform: none;
 `;
 
-const RadioGroup = styled(BaseRadioGroup)`
-  margin-top: 1rem;
-  .MuiFormGroup-root {
-    border: none;
-    margin-top: 0.5rem;
-    flex-direction: column;
-  }
-  legend {
-    color: ${({ theme }) => theme.palette.text.primary};
-    font-size: 1rem;
-    font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
-  }
-  label {
-    background: transparent;
-    border: none;
-    padding-top: 0;
-  }
-  .MuiSvgIcon-root {
-    width: 1.5rem;
-    height: 1.5rem;
-  }
-`;
-
-const Legend = styled.legend`
-  color: ${({ theme }) => theme.palette.text.primary};
-  font-size: 1rem;
-  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
-`;
-
-const Checkbox = styled(BaseCheckbox)`
-  margin: 0.5rem 0 0 1rem;
-  .MuiButtonBase-root {
-    padding: 0;
-    margin-right: 0.5rem;
-  }
-  label {
-    padding: 0.5rem 0 0 0.5rem;
-  }
-`;
-
 const RightColumn = styled.div`
   flex-grow: 1;
   width: 70%;
```

Comment:
I realise `grid` can be polarising (maybe?), so feel free to ignore, but I propose:

- Make `ExportContentContainer` `display: grid`
- On large displays, override the default single-column layout to `grid-template-columns: 3fr 7fr;`

Arguably nice to have the layout logic together, rather than separate in `LeftColumn` and `RightColumn`

Reply:
Good idea, have added, thanks!